governance/votes/Trial Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,191 @@
+
+The Governance Facilitator(s), Sidestream, dewiz, and PullUp have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- Multiple risk parameters will be changed including Stability Fees and Liquidation Ratios, as detailed below.
+- The Dai Savings Rate will be decreased to 3.19%.
+- DAI streams totaling 6,685,806 DAI will be set up for Chronicle Labs and Jet Stream development budgets.
+- MKR streams totaling 3836.33 MKR will be set up for Chronicle Labs and Jet Stream development budgets. 
+- Current streaming budget for DUX-001 will be canceled.
+- A total of 494,001 DAI will be transferred to the Jet Stream Ecosystem.
+- A total of 229.44 MKR will be distributed to previously active Aligned Delegates.
+- A total of 125 MKR will be transferred to the Deco Fixed Rate Core Unit.
+- A total of 56.8 MKR will be transferred to the Development & UX Core Unit (DUX-001). 
+- Legacy RWA Multiswap Conduit will be removed.
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- The FlapperUniV2 (Smart Burn Engine) will be added to replace Surplus Auctions (flaps).
+- Multiple Cron Jobs will be added to Chainlog
+- A new Multiswap Conduit will be added for RWA015-A.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Stability Scope Defined Parameter Changes
+
+As per the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104), and the [adjustments to the base rate](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238/6), the following parameter changes will be made, if this executive proposal passes:
+
+**[Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) Decrease**
+- Decrease the DSR by 0.30% from 3.49% to 3.19%
```

Comment:
You have a minor tense consistency issue here and in statements below. Generally speaking, these are things that _will_ happen if the executive proposal passes, so they should be written as future-tense as you have above: 'the following parameter changes will be made if this executive proposal passes'

This issue is also present in the official version...

The reason for this is to make very clear and avoid ambiguity or conflict around what you are voting for when you vote for the executive. Like, as a voter, you are not expressing an opinion at this stage on whether you want to reduce a stability fee parameter, instead you are contributing to the _execution_ of the change with your vote.